http/instances.go

```diff
@@ -29,23 +39,22 @@ func handleInstancesGet(ctx context.Context, core *cloudbrain.Core, w http.Respo
 	// Determine the path...
 	prefix := "/instances/"
 	if !strings.HasPrefix(r.URL.Path, prefix) {
-		respondError(ctx, w, http.StatusNotFound, nil)
+		respondError(ctx, w, http.StatusNotFound, errNoURLPrefix)
 		return
 	}
 	path := r.URL.Path[len(prefix):]
 	if path == "" {
-		respondError(ctx, w, http.StatusNotFound, nil)
+		respondError(ctx, w, http.StatusNotFound, errNoURLPath)
 		return
 	}
 
 	instance, err := core.GetInstance(ctx, path)
 	if err != nil {
-		cbcontext.LoggerFromContext(ctx).WithField("err", err).Error("couldn't get instance")
```

Comment:
Yeah, I agree that sending the error to sentry is probably a good idea in this case. I'll add a `cbcontext.CaptureError()` call.